### PR TITLE
Added DiExtra warmup problem fix documentation

### PR DESCRIPTION
**Pull request by Padam87**
#991

### CODE REVIEW

File: Resources/doc/reference/troubleshooting.rst [hunk 1]
```diff
@@ -106,3 +106,27 @@ Finally your settings in your ``sonata_media`` parameters will look like this:
...
+---------------------
+
+You may encounter the following error when warming up the cache if you don't have the admin bundle installed:
+
+.. code-block:: text
+
+    Fatal error: Class 'Sonata\AdminBundle\Controller\CRUDController'
+    not found in .....\vendor\sonata-project\media-bundle\Controller\GalleryAdminController.php on line 19
+    
+This problem is most likely caused by the JMSDiExtraBundle's cache warmer (https://github.com/schmittjoh/JMSDiExtraBundle/blob/master/HttpKernel/ControllerInjectorsWarmer.php).
```
Comment (greg0ire) [inline Resources/doc/reference/troubleshooting.rst]: Either `JMSDiExtraBundle's cache warmer` or `the JMSDiExtraBundle cache warmer`, but not both (I prefer the former). Also, try to keep lines under 120 characters.

File: Resources/doc/reference/troubleshooting.rst [hunk 2]
```diff
@@ -106,3 +106,27 @@ Finally your settings in your ``sonata_media`` parameters will look like this:
             local:
                 directory:  "%kernel.root_dir%/../web/uploads/media"
                 create:     false
+
+ 
+Error on cache:warmup
```
Comment (greg0ire) [inline Resources/doc/reference/troubleshooting.rst]: I would name this section "Usage without the admin bundle" (this might not be the last issue).

**Comment by greg0ire** [comment]
Now that I think of it, it might be possible to configure the JMS bundle with [this trick](http://symfony.com/doc/current/cookbook/bundles/prepend_extension.html)

If you're motivated, could you give it a look ? It would go that way : detect if the admin bundle is present, and if it isn't prepend the blacklist configuration. If you don't have the time, we'll probably merge this instead.

**Comment by Padam87** [comment]
Sorry, but this is it for me on this issue.

**Comment by core23** [comment]
This looks like a hack to solve this issue. Any possibility to fix this the proper way? Maybe we should fix ``JMSDiExtraBundle``.

**Comment by core23** [comment]
There is an issue https://github.com/schmittjoh/JMSDiExtraBundle/issues/126

**Comment by greg0ire** [comment]
> This looks like a hack to solve this issue. Any possibility to fix this the proper way? Maybe we should fix JMSDiExtraBundle.

Yeah and not every body uses The di extra bundle, (it's an extra bundle, after all), so maybe we should just merge this in the documentation, and that's all.

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**